srcpkgs/python3-s-tui/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-s-tui'
 pkgname=python3-s-tui
-version=1.1.3
-revision=3
+version=1.1.4
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools stress python3-urwid python3-psutil"
```

Comment:
what would you think about replacing stress with stress-ng?
```suggestion
depends="stress python3-urwid python3-psutil"
```

Reply:
> what would you think about replacing stress with stress-ng?

Seams reasonable IMO. From what I can see `stress-ng` is (more) actively developed (and feature-rich). 

I changed it to `stress-ng`.